java/src/jmri/jmrix/loconet/messageinterp/LocoNetMessageInterpret.java

```diff
@@ -1234,7 +1233,7 @@ private static String interpretOpcPeerXfer20(LocoNetMessage l) {
     }
 
     private static String interpretOpcPeerXfer20Sub10(LocoNetMessage l) {
-        /**
+        /*
```

Comment:
Is this change intended? It changes the comment from being Javadoc to an ordinary comment.

Reply:
I was wondering if it would make sense to move these JavaDocs from within the method to normal JavaDoc just above the method?  It may not!

Reply:
Thanks for remarks. Rolled back this change.